app/src/main/java/com/framgia/soundclound/screen/addtracktoalbum/AddTrackActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.framgia.soundclound.screen.addtracktoalbum;
+
+import android.content.Context;
```

Comment:
ERROR: Unused import: android.content.Context. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports